documentation/glossary.md

```diff
@@ -0,0 +1,56 @@
+# Glossary of aviation terms
+*Openscope uses some terms specific to aviation, this glossary attempts to define them.*
+---
+## ILS
+*abbr. of* Instrument Landing System
+
+A radio antenna at the end of the runway, known as a ''localizer'', guides pilots down onto the runway for landing, via an instrument known as a *guide scope*.
```

Comment:
This isn't exactly correct.

The localiser is used to line up with the runway for landing
The glideslope is used to descend an aircraft for landing

Reply:
done